src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let charge;
+
+  if (days < 3) {
+    charge = 0;
+  } else if (days < 7) {
+    charge = 20;
+  } else {
+    charge = 50;
+  }
```

Comment:
Move all of this 'magic numbers' into variables.
Also in this case it's better to create two separate if blocks and combine conditions with logical operators where it's possible